### ISSUE TITLE
Wait, can we even use CC BY-NC-SA dataset in StreetComplete?

### ISSUE THREAD

**Issue by matkoniecz** [rugk/crops-parser#60]
I started looking around as result of https://github.com/streetcomplete/StreetComplete/issues/3851 and spotted https://github.com/rugk/crops-parser#legal-stuff

Is it OK to use CC BY-NC-SA dataset in StreetComplete? Especially "noncommercial" part?

Is it ok to use it [GPLv3 software](https://github.com/streetcomplete/StreetComplete/blob/master/COPYING)? Is it OK to use it given https://github.com/streetcomplete/StreetComplete#sponsors ?

**Comment by rugk** [comment]
Yeah also stumbled upon this since the last update and I guess this is new that they have added that license, so I reproduced it here.

GPLv3… could only be a problem if there is a license incompatibility. [Official documents only tell about SA-licenses and compatibility with GPL](https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses), which of course makes sense as SA is like copyleft in a GPL license end evaluate the compatibility.

> Especially "noncommercial" part?

https://creativecommons.org/faq/#does-my-use-violate-the-noncommercial-clause-of-the-licenses

It's all about legal interpretation, as it seems…
And also about the jurisdiction, which – in our case – [would be some international "one"(?)](https://wiki.creativecommons.org/wiki/Intergovernmental_Organizations#What_should_I_know_before_I_use_a_work_licensed_under_the_IGO_3.0_ported_licenses.3F), i.e. and arbitral tribunal could only decide that… after all it's data from an international organization, that kinda makes sense. (also did not know such special CC-licenses exist, only discovered it here).

I personally would say that as long as your aim is not to collect money with what you do (no advertising etc.), this is not a commercial purpose. After all, it's defined as…
> not primarily intended for or directed towards commercial advantage or monetary compensation.

"Primarily" is important here and for all other discussions I [again can only point to the CC's own explanation](https://wiki.creativecommons.org/wiki/NonCommercial_interpretation).

As they say there though, one could maybe always just ask for clarification:
> If you are unsure, you should either contact the rights holder for clarification, or search for works that permit commercial uses.

After all, of course only they could enforce their copyright there, anyway, if they deemed the usage to be "incompliant".

What we could do to be more compliant (with the BY section) would show [that one sentence here in the About section of StreetComplete](https://github.com/rugk/crops-parser/blob/master/LICENSE-data.md#fao):
> FAO.QCL. License: CC BY-NC-SA 3.0 IGO. Extracted from: https://www.fao.org/faostat/en/#data/QCL. Data of Access: 14-02-2022

(That's what FAO says how they want to be cited. And yes, they want that crazy data format, I also think that's crazy… :upside_down_face:)

----

That all said IANAL – I am not a lawyer, so these are just guesses here and of course the final word/responsibility is @westnordost's one…

**Comment by matkoniecz** [comment]
> Yeah also stumbled upon this since the last update and I guess this is new that they have added that license, so I reproduced it here.

Hmm, I wonder what was there earlier.

And is this extract of dataset used by SC significant enough to be considered as covered by copyright.

**Comment by rugk** [comment]
> And is this extract of dataset used by SC significant enough to be considered as covered by copyright.

I have no idea, yet another legal question… I guess the original data certainly meets [the threshold of originality](https://en.wikipedia.org/wiki/Threshold_of_originality), which – as we use that part of the data in the end – extends to all stuff where it is used. IMHO. IANAL, again.

> Hmm, I wonder what was there earlier.

I could have simply overlooked it… just saying. However [the first web captures of that license page begin in 2020](https://web.archive.org/web/20200401000000*/https://www.fao.org/contact-us/terms/db-terms-of-use/en/) (and are pretty regular)…
Maybe they just did not provide any license information?

Unfortunately the [source data website could not be captured properly](https://web.archive.org/web/20170201023139/http://www.fao.org/faostat/en/) likely because it is a huge bunch of JS/interactively generated and Archive.org misses the `#` client information passed in there.
